packages/@okta/vuepress-site/docs/guides/add-an-external-idp/use-idp-to-sign-in/index.md

```diff
@@ -6,10 +6,12 @@ To test your authorization URL, enter the complete authorization URL in a browse
 If everything is configured properly:
 
 * The user is redirected to the Identity Provider's sign-in page.
-* After successful authentication, the user is redirected to the redirect URI that you specified, along with an `#id_token=` fragment in the URL. The value of this parameter is your Okta OpenID Connect ID token.
+* After successful authentication, the user is redirected to the redirect URI that you specified, along with an `#id_token=` fragment in the URL. The value of this parameter is your Okta OpenID Connect ID token. 
 
 If something is configured incorrectly, the authorization response contains error information to help you resolve the issue.
 
+> **Note:** With Sign In with Apple, when a user signs in, Apple offers the user the option to either share their email address or hide it. If the user chooses to hide their email address, Apple generates a random email address and sends that to Okta. The user must choose the **Share My Email** option if you want to link the user's Apple account to an existing account in Okta.
```

Comment:
...Apple generates a random email address and sends that to Okta. However, Apple maintains the mapping between this random email address to the user's original email address. The user must choose the **Share My Email** option if you want to link the user's Apple account to an existing account in Okta. A user can choose this option only when the user is signing in with Apple against Okta for the first time.